elated-egrets/bangazon-internal-site-2.0
# Rm add product to order

## DESCRIPTION
This pr is a partial reference to #32 
it has the order model required for the rest of the code

## How to pull down
Commands

```
git fetch --all
git checkout rm-add-product-to-order
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. make new migrations
1. go to database and make sure the order and order_products model tables have been created

## What you changed
added the order model

## What was added to read me
info on the order model

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Have 2 teammates approved the PR?

Then Ship It!
